distributions/openhab-online/readme.md

```diff
@@ -0,0 +1,37 @@
+openHAB online distro
+---------------------
+
+
+The online distro doesn't contain many features.
+You'll need to install those manually for now.
+
+
+1. start openhab
+	start.sh
+
+2. view available features
+	feature:list
+
+3. install a feature e.g. install rest docs
+	feature:install openhab-misc-restdocs
```

Comment:
> I'm not sure what you mean with automatically constructed.

Note that all of this is in place with the addon.cfg already. I'll enhance this with the "runtime" setting tomorrow, so you don't have to worry about this now in the context of this PR. For the distribution, it would therefore be enough if you simply have the feature openhab-runtime-base pre-installed - ok?